src/util/fields.py

```diff
@@ -833,6 +833,7 @@ def _update_field_usage_multi(cr, models, old, new, domain_adapter=None, skip_in
 
     p = {
         "old": r"\y%s\y" % (re.escape(old),),
+        "old_pattern": r"""\.{0}\y|"{0}"|'{0}'""".format(re.escape(old)),
```

Comment:
Matching the ending quote will ignore patterns like `.filtered("foo_id.bar")`.
```suggestion
        "old_pattern": r"""[.'"]{0}\y""".format(re.escape(old)),
```

Reply:
Same I proposed [here](https://github.com/odoo/upgrade-util/pull/29#discussion_r1434221635)

I agree it would be enough...